Project/jobportal/scripts/signup.js

```diff
@@ -0,0 +1,48 @@
+document
+  .getElementById("registrationForm")
+  .addEventListener("submit", (event) => {
+    event.preventDefault();
+    var fullname = document.getElementById("fullname").value;
+    var email = document.getElementById("email").value;
+    var password = document.getElementById("password").value;
+    var confirm_password = document.getElementById("confirm_password").value;
+
+    if (password !== confirm_password) {
+      alert("Passwords do not match");
+      return;
+    }
+
+    var requestBody = {
+      name: fullname,
+      email: email,
+      password: password,
+    };
+
+    fetch("http://127.0.0.1:8000/api/register", {
```

Comment:
@codal-kcolaco please take this as `http://127.0.0.1:8000/api` as const variable and use it everywhere because if the host changes then you will have to change it in one place only